docs(chart): Add kb about how to apply gradient color effect to area … (#1257)
telerik/blazor-docs: knowledge-base/chart-area-gradient-color.md

---
title: Apply gradient effect on Area Chart
description: How to apply gradient as color for Area Chart series?
type: how-to
page_title: Apply gradient effect on Area Chart
slug: chart-area-gradient-color
position: 
tags: chart, area, gradient, color

> ticketid: 1591373

Reviewer: There is an older ticket about this scenario too.

```suggestion
ticketid: 1545645, 1591373
```

res_type: kb
---

## Environment
<table>
	<tbody>
		<tr>
			<td>Product</td>
			<td>Chart for Blazor</td>
		</tr>
	</tbody>
</table>

## Description
How can I apply a gradient color effect on the Telerik UI for Blazor Area Chart?

## Solution

> 1. Declare an external gradient.

Reviewer: This step is not actionable and does not bring value. Provide a few more words how to define the gradient. There is no problem to link official third-party documentation for general knowledge.

```suggestion
1. Declare an external [linear gradient with an `svg` tag](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/linearGradient).
```

> 1. Use the [Color](https://docs.telerik.com/blazor-ui/components/chart/types/area#color) configuration of the Chart series to set the external gradient. 

Reviewer: ```suggestion
1. Use the [Color](https://docs.telerik.com/blazor-ui/components/chart/types/area#color) parameter of the Chart series to set the external gradient via its `id`. 
```

>caption How to apply gradient effect on Area Chart.

````CSHTML
@*Gradient*@
<div style="height: 0">
    <svg xmlns="https://www.w3.org/2000/svg" version="1.1" width="0" height="0" style="visibility: hidden">
        <defs>
            <linearGradient id="svg-gradient" x1="0%" y1="0%" x2="0%" y2="100%">
                <stop offset="0%" style="stop-color:#3bafda; stop-opacity:0.95" />
                <stop offset="100%" style="stop-color:#3bafda; stop-opacity:0.05" />
            </linearGradient>
        </defs>
    </svg>
</div>

> 
> @*Chart*@

Reviewer: Not necessary.

```suggestion
```

> <TelerikChart Class="chart">

Reviewer: Do we need the `Class`? Try to remove anything that is not relevant to the example (except for required real-world settings that may force developers to add them back to make the scenario meaningful).

```suggestion
<TelerikChart>
```

    <ChartSeriesItems>
        <ChartSeries Type="ChartSeriesType.Area"
                        Color="url(#svg-gradient)"
                        Data="@SimpleData">
        </ChartSeries>
    </ChartSeriesItems>
    <ChartValueAxes>
        <ChartValueAxis Color="#ffffff00">
            <ChartValueAxisLabels Color="#A4A4A4"></ChartValueAxisLabels>
        </ChartValueAxis>
    </ChartValueAxes>
    <ChartCategoryAxes>
        <ChartCategoryAxis Categories="@XAxisItems">
            <ChartCategoryAxisLabels Color="#A4A4A4"></ChartCategoryAxisLabels>
        </ChartCategoryAxis>

>     </ChartCategoryAxes>

Reviewer: This configuration is not relevant to the main focus of the example.
